cmd/tiller/tiller.go

```diff
@@ -90,44 +102,102 @@ func main() {
 }
 
 func start(c *cobra.Command, args []string) {
+	// Set the log level first
+	logLevel, err := logutil.GetLevel(level)
+	if err != nil {
+		logger.WithFields(log.Fields{
```

Comment:
Maybe you could consider moving most commonly used log.Fields to a variable within start function (and other functions?

I.e. you can just define at the beginning of `start`
```
logFields := log.Fields{
  "_module": "tiller",
  "_context": "start",
}
```
When no additional info is needed (`error`, `level` fields etc.) just use:
`logger.WithFields(logFields).Error(...)`

When additional info is needed:
```
f := logFields
f.error = err
logger.WithFields(f).Error(...)
```

Seems like this way you could shave off some of redundant log fields definitions, especially in longer functions (like `start`)